src/home/diagnostictool/DiagnosticTool.js

```diff
@@ -59,28 +78,34 @@ export default class DiagnosticTool extends React.Component<DropdownIsVisibleSta
     return (
       <View style={styles.container}>
         <NavHeader title="Diagnostic Tool" {...{ navigation }} />
-        <LinearGradient colors={["#ffffff", "#ffffff"]} style={styles.gradient} />
+        <LinearGradient colors={["#81f1f7", "#9dffb0"]} style={styles.gradient} />
```

Comment:
this line needs to be removed completely, we are currently not using LinearGradient, and be sure to remove the import at the top of the file as well